docker_entrypoint.sh

```diff
@@ -2,13 +2,36 @@
 
 set -e
 
+# Get the architecture of the host
+ARCH=$(uname -m)
 
 while [ ! -f /jellyfin/main/start9/config.yaml ]; do
     sleep 1
 done
 
 echo "File /jellyfin/main/start9/config.yaml has been created."
 
+grep -q "filebrowser" /jellyfin/main/start9/config.yaml && FILEBROWSER=true || FILEBROWSER=false
+grep -q "nextcloud" /jellyfin/main/start9/config.yaml && NEXTCLOUD=true || NEXTCLOUD=false
```

Comment:
This seems a bit brittle, as it is not robust against adding the word "filebrowser" or "nextcloud" anywhere in the config file. you should use `yq` to select the specific value that affects this behavior.

Reply:
This has been updated to use `yq` as suggested in cec7cfaec76f322a451bb847e11201aa0456d614